main.py

```diff
@@ -31,4 +31,4 @@ def convert():
 
 
 if __name__ == '__main__':
-    app.run(port=8000)
+    app.run(port=8000, host='0.0.0.0')
```

Comment:
We need to specify the host explicitly for it to work inside of a docker image (this is functionally the same, it can still be reached at 127.0.0.1, but makes dockers networking happy)